.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:
+
+- [ ] Branch rebased on latest master `git pull --rebase upstream master`
+- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
+- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
+- [ ] Commit message includes `Fixes #<id>` where `<id>` is the associated MantisBT issue id if one exists
```

Comment:
Line 6: Maybe we should have people read and understand https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT because they can also [link|fix] [single|multiple] fixes ?